internal/api/api.go

```diff
@@ -36,13 +36,15 @@ func IsContainerCertified(registry, repository, tag, digest string) bool {
 }
 func IsOperatorCertified(operatorName, ocpVersion, channel string) bool {
 	if onlineClient.IsServiceReachable() {
+		logrus.Debug("Online service is reachable")
```

Comment:
Do you really need this extra log line? There's already a logrus.Warnf (lines 42 and 50) in case oneClient.IsServiceReachable() returned false.